### ISSUE TITLE
errno 13 permission denied on log file

### ISSUE THREAD

**Issue by rahh1861** [giuspen/cherrytree#1724]
**Version, Operative system**
Windows 10; Cherrytree 0.99.36 and 0.99.38

**Describe the bug**
Hello. I am a user of Cherrytree. I have a problem that was reported in the Github area several years ago and is now marked as closed - #416

When I run Cherrytree in the regular manner (from the taskbar), I get the error:

"Errno 13:
logfile c:\Program Files (x86)\bin\Cherrytree.exe.log could not be opened: permission denied on c:\Program Files (x86)\bin\Cherrytree.exe.log"

and the program stops running.

So I tried running it as administrator. When I did so, I got the following error:

"errors occurred, see c:\Program Files (x86)\bin\Cherrytree.exe.log for details." 

Then the program stops running.

This was using Cherrytree version .99.36 . I downloaded the new .99.38 and installed it, but get the same errors.

The logfile from running it as administrator has this:
-------------

Traceback (most recent call last):
  File "cherrytree", line 78, in <module>
  File "cherrytree", line 50, in f_main
  File "main.pyc", line 248, in main
  File "main.pyc", line 83, in __init__
  File "main.pyc", line 88, in window_open_new
  File "core.pyc", line 252, in __init__
  File "core.pyc", line 1500, in file_startup_load
  File "core.pyc", line 2010, in file_load
  File "support.pyc", line 1776, in set_bookmarks_menu_items
ValueError: invalid literal for long() with base 10: ''

------------

Unfortunately, this is on a computer at a museum I volunteer at, so I cannot do any easy testing for you (it runs fine on my home computer, which I am using now). Both computers are running the latest Windows 10.

I do think it would be good if you could let the program run even when it gets such an error, if at all possible. I mean, it absolutely stopped me cold in my tracks when I was at this museum, since I needed info contained in the Cherrytree databases to do the work. I managed to get by using an old version of Treepad which had a database with obsolete but usable info. But in my opinion this is not an optimum way to have Cherrytree work. 

I mean, I don't care about the logfile (usually), and just want to be able to look at my database. I understand that it's not a good idea to be using a program that is having trouble opening its files, but well, it would be good if you could alter the way it works somehow. Perhaps make the logfile optional in the Preferences?

Anyway, I am unable to use Cherrytree at that museum till a fix is available. Otherwise I really like your program and appreciate it very much. It is a very good replacement for Treepad and in some ways is even better. :-)

Rich Hureau

**Comment by giuspen** [comment]
The problem here is that the old cherrytree is actually running. Try to uninstall and reinstall again. To prove that you are not really running the latest version try to download the portable archive, extract and double click on cherrytree.exe there.

**Comment by rahh1861** [comment]
I think that the logfile that I put on a flash drive to take home with me (to use for editing this Github post) was from the earlier running of the earlier 99.36 CT. But I DID install the newer version and tried it and it still had the same problem. Unfortunately, I am not able to get to the museum for a while. I will do so, but it might be awhile (I cannot drive because I broke my collarbone :( )

**Comment by giuspen** [comment]
c:\Program Files (x86)\bin\Cherrytree.exe is the path of the old 32 bit python version, you won't have the (x86) in the path now since it's 64 bit.

**Comment by rahh1861** [comment]
Yes, I thought that seemed odd. I did notice that I had BOTH 64 and 32 folders for CT on that computer. I just figured that's the way it worked. Hmm. Perhaps I was using an old icon on the taskbar that points to an old version. I will check that. Glad you noticed that! I will let you know asap. :)

**Comment by rahh1861** [comment]
The more I think about this, I think you have definitely identified the problem - I am running an old version, even though I installed the newest version. 

That computer in the museum hasn't been used in over a year (because of the pandemic), and I went into the museum and loaded a bunch of CT files from home that had been made/updated in the latest version of CT at home. Then I ran CT in the museum and had that problem, using an icon sitting on the taskbar. I did then install the new version (but a 64-version, which did NOT update that taskbar icon's target exe), and then used the same icon to run it, and no doubt picked up the earlier version. Yikes!!

So, good detective work! I have closed this (with apologies!).

Rich
